client/my-sites/upgrades/checkout/style.scss

```diff
@@ -1,169 +1,3 @@
-.checkout__purchase-detail-description {
```

Comment:
For some reason, this stylesheet wasn't included from `assets/_components.scss`, and contained a bunch of old styles.